templates/e2e-llm-workflows/README.ipynb

```diff
@@ -677,20 +677,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
    "id": "85db636c",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "name: llm-fine-tuning-guide\n",
-      "entrypoint: python src/ft.py configs/training/lora/llama-3-8b.yaml\n",
-      "image_uri: localhost:5555/anyscale/llm-forge:0.4.3.2\n",
+      "name: e2e-llm-workflows\n",
+      "entrypoint: llmforge anyscale finetune configs/training/lora/llama-3-8b.yaml\n",
```

Comment:
did you verify that this will directly substitute https://github.com/anyscale/templates/blob/e19a41fb40d0be625d19c4d204a5196e87a64d98/templates/e2e-llm-workflows/src/ft.py?

Reply:
The code in `src/ft.py` has been upstreamed into llmforge quite some time back- I thinbk in our 0.5.1 release. 

To verify, I actually launched a fine-tuning job - and retrieved model ID from there - you can see the execution output for `anyscale.llm.model.get` 


Technically the code block can look like:
```
import anyscale
from anyscale.job.models import JobConfig, JobState

job_id: str = anyscale.job.submit(
    JobConfig(
        name="llmforge-fine-tuning-job",
        entrypoint="llmforge anyscale finetune config.yaml",
        working_dir=".",
        image_uri="localhost:5555/anyscale/llm-forge:0.5.4"
    ),
)
# Wait until job succeeds, with a 5 hour timeout for the wait operation. See the API reference for more details: https://docs.anyscale.com/reference/job-api 
anyscale.job.wait(id=job_id, timeout_s=18000)
model_info = anyscale.llm.models.get(job_id=job_id)
```


I didn't add the `anyscale.job.wait` thing because it's meant to be used with the Job SDK , while here we're using a YAML for the job.